server/routes.go

```diff
@@ -33,6 +33,7 @@ func DoRoutes() chi.Router {
 		r.With(identity.EnforceIdentity).Route("/", controllers.LubDub)
 		r.Route("/openapi.json", apispec.OpenAPISpec)
 		r.With(identity.EnforceIdentity).Get("/services", controllers.Index())
+		r.With(identity.EnforceIdentity).Get("/compliance", controllers.Compliance())
```

Comment:
We should update the [openapi.json spec](https://github.com/RedHatInsights/entitlements-api-go/blob/master/apispec/api.spec.json) to include this new endpoint and what the possible responses from the service would look like, especially since we'll get a 200 for 👍 and 👎:
```
{"result":"OK","description":""}
{"errors":[{"error":"no_such_user","identityType":"login","identity":"foo@redhat.com"}]}
```

Reply:
also updated to return a 400 on bad request cuz compliance actually returns a 400 when user not found. I updated the endpoint to pass the status code that compliance returned instead of always doing a 200 on a successful call